airflow/executors/kubernetes_executor.py

```diff
@@ -297,7 +297,7 @@ def run_next(self, next_job: KubernetesJobType) -> None:
             kube_image=self.kube_config.kube_image,
             try_number=try_number,
             date=execution_date,
-            command=command,
+            args=command,
```

Comment:
This needs a note in the UPDATING.md

Reply:
Do we need to mark this if it's not a publicly advertised feature?

Reply:
It changes how the pod runs, so anyone relying on the old behaviour might get bit

Reply:
@ashb but no one relies on the old behavior. This is not a feature that people would use when building DAGs. It only works in the context of adding in stuff specific to the KubernetesExecutor.

Reply:
Ohhh ok I see what you're saying. This affects a user who would say... modify the command and args section of the pod in the pod_mutation_hook.

Reply:
@ashb in that case do we really want to break this?

Reply:
It's not about pod mutation hook - but about _how the pod is run_.

Previously this did the effect of `docker run --entrypoint airflow ... run`

Now it runs the entrypoint (which is more correct) and needed to actually fix that bug, so yes, we want this, but just need to include a note about it in the docs.

Reply:
@ashb but this WILL break pod_mutation_hooks

example:

```
def pod_mutation_hook(pod):
    if "foo" in pod.spec.containers[0].command
    ....
```

Reply:
True.

So 2.0 is the time to do it :D

Reply:
@ashb fair enough :). I'll add this, add to the UPDATING, and then add an upgrade rule after.

Reply:
Upgrade rule may be tricky for this one. Ace of you can do it, but we can't get a rule for everything